test/scripts/iou.sh

```diff
@@ -28,7 +30,7 @@ else
 fi
 
 # install chaincode on org1
-CC_PATH="${TEST_ROOT}/models/iou/hlf/src/iou"
+CC_PATH="${TEST_ROOT}/models/iou/iou/src/iou"
```

Comment:
Yes, but it is better to push that decision to the final user using the -t target flag instead of harcoding the name. 

This way the user can generate as many apps as json files in the same folder as long and the name in the json file is different.

Reply:
ok.  it is fine as long as the second "iou" in the above script is user provided folder.

Reply:
Yes, both are coming from the json file, so both are user provided